### PR TITLE
CloudSQL new rule: cloudsql/BP/2023_001

### PR DESCRIPTION
**Pull request by abhigupta1207**
Added a new rule for CloudSQL for public IP

### CODE REVIEW

File: gcpdiag/lint/gke/snapshots/BP_2022_003.txt
Comment (junggil) [inline gcpdiag/lint/gke/snapshots/BP_2022_003.txt:1]: I guessed that this snapshot is not related with the PR and should not be changed.  
Could you take a look?

**Review by junggil — COMMENTED** [review]
Thanks for the contribution!  
All looks good except one minor thing. Just left a comment, PTAL.

File: gcpdiag/lint/gke/snapshots/BP_2022_003.txt
Comment (abhigupta1207) [reply gcpdiag/lint/gke/snapshots/BP_2022_003.txt:1]: @junggil Unstaged the file

**Comment by abhigupta1207** [comment]
@junggil unstaged the file

**Comment by junggil** [comment]
Thanks!
